src/api/kotlin/network/rs485/logisticspipes/property/Property.kt

```diff
@@ -42,6 +42,8 @@ import java.util.concurrent.CopyOnWriteArraySet
 
 interface Property<V> : IStore {
     val tagKey: String
+    val oldTagKey: String
+        get() = ""
```

Comment:
I think adding this to an interface and exposing backwards-compatibility this way doesn't make sense. The issue with backwards-compatibility should be an implementation detail, instead of duplicating the tag key on the interface. That way the code for backwards-compatibility is much safer to delete when it has gotten unnecessary.

Reply:
So put it on the one that required should be good?

Reply:
I would be happy with something like this: https://github.com/RS485/LogisticsPipes/blob/e59c3c674e33a0979ecaab9a9c5e0a035833d9d9/common/logisticspipes/modules/ModuleCrafter.java#L702-L741
That is the backwards-compatible NBT loading code for the crafting module.

Reply:
> I think adding this to an interface and exposing backwards-compatibility this way doesn't make sense. The issue with backwards-compatibility should be an implementation detail, instead of duplicating the tag key on the interface. That way the code for backwards-compatibility is much safer to delete when it has gotten unnecessary.

Oh, the old key won't be saved to the NBT data
Its only to convert back to new tag key